float32 joint1
float32 joint2
float32 joint3 
float32 joint4
float64 arm_palm
float64 palm_finger
float64 finger_finger_tip